### DISCUSSION TITLE
[V3] New Documentation x Nested Menu in Mobile.

### DISCUSSION THREAD

**Discussion by devajmeireles** [livewire/livewire#5839] [7. Bugs (archived)]
Hey, folks!

The new site looks amazing, it was what Livewire had been missing for a long time, but I realized that two things were missing.

1. When we are browsing on a mobile device, we cannot see the menu option: _"ON THIS PAGE"_ nested within a page.

2. We don't have a search box to search documentation indexes.

Hope this help,

Thankssss.

**Comment by joshhanley** [comment]
@devajmeireles thanks! We'll get someone to have a look.
